backend-project/small_eod/institutions/serializers.py

```diff
@@ -4,13 +4,16 @@
 )
 from rest_framework import serializers
 from ..generic.serializers import UserLogModelSerializer
+from ..tags.serializers import TagSerializer
 
 
 class InstitutionSerializer(UserLogModelSerializer):
     administrative_unit = serializers.PrimaryKeyRelatedField(
         many=False, queryset=JednostkaAdministracyjna.objects.all(),
     )
 
+    tags = TagSerializer(read_only=True, many=True)
```

Comment:
To nie może być tylko do odczytu, bo będzie to podstawowa forma dodawania tagów. Zobacz jak rozwiązano ten problem w `CaseSerializer`.